SwanCustomEnvironments/swancustomenvironments/serverextension.py

```diff
@@ -22,16 +22,16 @@ def get(self):
         """
         self.set_header("Content-Type", "text/event-stream")
 
-        repository = self.get_query_argument("repo", default=None)
-        repository_type = self.get_query_argument("repo_type", default=None)
-        accpy_version = self.get_query_argument("accpy", default=None)
+        repository = self.get_query_argument("repo", default='')
+        repository_type = self.get_query_argument("repo_type", default='')
+        accpy_version = self.get_query_argument("accpy", default='')
 
         arguments = ["--repo", repository, "--repo_type", repository_type]
-        if accpy_version is not None:
+        if accpy_version != '':
```

Comment:
If you know `accpy_version` is always going to be a string, you can do:
```
if accpy_version:
```
instead to check if it's length is greater that zero.